src/+otp/+pendulum/f.m

```diff
@@ -0,0 +1,6 @@
+function dy = f(~, y, g, l)
+dy = [
```

Comment:
I'm not sure that I like the spacing here.

Reply:
Also, support for vectorization is trivial: replace y(1) with y(1, :) and y(2) with y(2, :).